test/unit/org/apache/cassandra/config/EncryptionOptionsEqualityTest.java

```diff
@@ -42,13 +42,13 @@ private EncryptionOptions.ServerEncryptionOptions createServerEncryptionOptions(
     {
         EncryptionOptions.ServerEncryptionOptions.Builder serverEncryptionOptionsBuilder = new EncryptionOptions.ServerEncryptionOptions.Builder();
         return serverEncryptionOptionsBuilder
+               .withOutboundKeystore(TlsTestUtils.SERVER_OUTBOUND_KEYSTORE_PATH)
```

Comment:
@smiklosovic Why is this order change necessary?

Reply:
@maulin-vasavada Because you dont have `withOutboundKeystore` on super class builder. If I used different methods, e.g. the ones on super class builder, then their methods would return me super class builder again, which do not have `withOutboundKeystore` on it. When I move `withOutboundKeystore` as the first, it will still return me the subclass builder until it does not when I use methods just found in the super one.